zdm/tests/mc_statistics.py

```diff
@@ -0,0 +1,386 @@
+
+"""
+This script illustrates how to generate MC samples and evaluate them
+Note that in this case, only MC samples for:
+- one survey (ASKAP fly's eye)
+- evaluated at the same parameters as they are generated
+- without varying the detected number of bursts
+is generated.
+
+In general, one will want to generate the samples with one set of parameters,
+and then evaluate them with another set of grids.
+Hence the saving of 'mc_sample.npy'
+
+"""
+
+import argparse
+
+import numpy as np
+from zdm import zdm
+#import pcosmic
+import matplotlib.pyplot as plt
+import matplotlib.colors as colors
+import matplotlib.cm as cm
+import matplotlib
+from pkg_resources import resource_filename
+import os
+import sys
+
+import scipy as sp
+import time
+from matplotlib.ticker import NullFormatter
+from zdm import iteration as it
+
+from zdm import survey
+from zdm import cosmology as cos
+from zdm import beams
+
+import pickle
+import copy
+
+matplotlib.rcParams['image.interpolation'] = None
+
+defaultsize=14
+ds=4
+font = {'family' : 'normal',
+        'weight' : 'normal',
+        'size'   : defaultsize}
+matplotlib.rc('font', **font)
+
+##### definitions of parameters ########
```

Comment:
After this PR, I am going to refactor the code so that the parameters
are handled with a highly interpretable `pset`.  e.g. a dict!

Reply:
OK, yes, dict it is!